site-docs/stylesheets/extra.css

```
@@ -1,4 +1,5 @@
 /* Synapse A2A - Custom Styles */
+@import url("palette.css");
```

Stylelint (17.4.0)

[error] Expected "url("palette.css")" to be ""palette.css"" (import-notation)

(import-notation)